lib/improver/tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -288,112 +288,59 @@ def test_callable(self):
         msg = ('<BaseNeighbourhoodProcessing: neighbourhood_method: '
                '<CircularNeighbourhood: weighted_mode: True, '
                'sum_or_fraction: fraction>; '
-               'radii: 10000.0; lead_times: None; ens_factor: 1.0>')
+               'radii: 10000.0; lead_times: None>')
         self.assertEqual(result, msg)
 
     def test_not_callable(self):
         """Test that the __repr__ returns the expected string."""
         result = str(NBHood("circular", 10000))
         msg = ('<BaseNeighbourhoodProcessing: neighbourhood_method: '
-               'circular; radii: 10000.0; lead_times: None; ens_factor: 1.0>')
+               'circular; radii: 10000.0; lead_times: None>')
         self.assertEqual(result, msg)
 
 
-class Test_adjust_nsize_for_ens(IrisTest):
-
-    """Test adjusting neighbourhood size according to ensemble size."""
-
-    def test_basic_returns_float(self):
-        """Test returns float."""
-        neighbourhood_method = CircularNeighbourhood()
-        radii = 20.0
-        ens_factor = 1.0
-        num_ens = 3.0
-        width = 20.0
-        result = NBHood(
-            neighbourhood_method, radii,
-            ens_factor=ens_factor).adjust_nsize_for_ens(num_ens, width)
-        self.assertIsInstance(result, float)
-
-    def test_returns_unchanged_for_ens1(self):
-        """Test returns unchanged value when num_ens = 1.0."""
-        expected = 20.0
-        neighbourhood_method = CircularNeighbourhood()
-        radii = 20.0
-        ens_factor = 0.8
-        num_ens = 1.0
-        width = 20.0
-        result = NBHood(
-            neighbourhood_method, radii,
-            ens_factor=ens_factor).adjust_nsize_for_ens(num_ens, width)
-        self.assertAlmostEqual(result, expected)
-
-    def test_returns_adjusted_values(self):
-        """Test returns the correct values."""
-        expected = 9.2376043070399998
-        neighbourhood_method = CircularNeighbourhood()
-        radii = 20.0
-        ens_factor = 0.8
-        num_ens = 3.0
-        width = 20.0
-        result = NBHood(
-            neighbourhood_method, radii,
-            ens_factor=ens_factor).adjust_nsize_for_ens(num_ens, width)
-        self.assertAlmostEqual(result, expected)
-
-
 class Test__find_radii(IrisTest):
 
     """Test the internal _find_radii function is working correctly."""
 
     def test_basic_float_cube_lead_times_is_none(self):
-        """Test _find_radii returns a float with the correct value."""
+        """
```

Comment:
We normally format doc-strings in tests without the line break after the quotes, e.g.

```
"""Test _find_radii returns an unaltered radius if
the lead times are none, and this radius is a float."""
```

Reply:
Changed.